specifics/Merlin/MerlinClass.cpp

```diff
@@ -720,7 +720,7 @@ void MerlinClass::attribute_factory(vector<Tango::Attr *> &att_list)
 	//	Attribute : gain
 	gainAttrib	*gain = new gainAttrib();
 	Tango::UserDefaultAttrProp	gain_prop;
-    gain_prop.set_description("0 = SHGM\n1 = HGM\n2 = LGM\n3 = SLGM");
+    gain_prop.set_description("0 = SLGM \n1 = LGM\n2 = HGM\n3 = SHGM");
```

Comment:
I still don't understand why this commit appear there ... I hope that for the next commit, all the stuff from Hamamatsu will not be there... (did you do a pull before starting to work ?)